### ISSUE TITLE
date formatting & sorting

### ISSUE THREAD

**Issue by sangermann** [waningflow/vue-virtual-table#26]
Hi,
thank you very much for your great work - this vue component truely rocks!
Is there a way to use a date column with formatted cell display (without template slot-scope) so that it is still sortable?

**Comment by waningflow** [comment]
You can format the date to string like `YYYY-MM-DD` and set the corresponding prop config with `sortable: true`.

**Comment by sangermann** [comment]
Yes, but I need it formatted with `DD.MM.YYYY` and/or `DD.MM.YYYY, HH:mm:ss`. At the moment this doesn't work correctly with `sortable: true`. Is there an alternative option to make it work?

**Comment by waningflow** [comment]
Currentlly no. It needs addition work.

**Comment by sangermann** [comment]
Maybe as new feature for the next release? :) I would donate for that because it's only useful for me with this functionality.

**Comment by sangermann** [comment]
YEAH! Thank you so much.
Now custom slot works great with formatted data and sorting!
Donated :)
